requirements.txt

```diff
@@ -0,0 +1,3 @@
+reframe

```

Comment:
I know it's still a draft, but just a small comment. reframe is actually a different package, you need to install ReFrame-HPC. And actually I just remembered that Jg wrote last week some docs about installing reframe with pip, to make it easier. https://confluence.cscs.ch/display/reframe/Installing+ReFrame
But I take your point, maybe we should have the requirements on the main directory of the repo, not in utilities

Reply:
Thanks, updated!

Reply:
Why is the installing e debug documentation kept private in confluence? Should I remove the section I have added to the readme file?

Reply:
I think it makes sense to have this information in the repository, you are right!